Temporary hack to avoid the mmsg structs, available only in the Linux socket.h.
A better fix (making them and a sendmmsg stub available for every OS) is going
to be done upstream.

--- a/resolv/res_send.c
+++ b/resolv/res_send.c
@@ -1109,6 +1109,7 @@ send_dg(res_state statp,
 		if (have_sendmmsg >= 0 && nwritten == 0 && buf2 != NULL
 		    && !single_request)
 		  {
+#ifdef __ASSUME_SENDMMSG
 		    struct iovec iov[2];
 		    struct mmsghdr reqs[2];
 		    reqs[0].msg_hdr.msg_name = NULL;
@@ -1161,6 +1162,10 @@ send_dg(res_state statp,
 			Perror(statp, stderr, "sendmmsg", errno);
 			goto err_out;
 		      }
+#else
+		    have_sendmmsg = -1;
+		    goto try_send;
+#endif
 		  }
 		else
 		  {
